### PR TITLE
Converted LoggerTest to Kotlin

### PR DESCRIPTION
**Pull request by Jessica-Kuo**
Fixed styling on some functions

### CODE REVIEW

File: analytics/src/test/java/com/segment/analytics/LoggerTest.kt [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        Assertions.assertThat(ShadowLog.getLogs())
+                .containsExactly(
+                       LogItemBuilder()
+                                .type(Log.VERBOSE)
+                                .msg("some message with an argument")
+                                .build())
+    }
+
+    @Test
+    fun debugMessagesShowInLog(){
```
Comment (migs647) [inline analytics/src/test/java/com/segment/analytics/LoggerTest.kt:40]: Nitpick: Try and keep your style and spacing consistent. In this case there should be a space between ) and {

`fun debugMessagesShowInLog() {`

File: analytics/src/test/java/com/segment/analytics/LoggerTest.kt [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        Assertions.assertThat(ShadowLog.getLogs())
+                .containsExactly(
+                LogItemBuilder()
+                        .type(Log.DEBUG)
+                        .msg("some message with an argument")
+                        .build())
+    }
+
+    @Test
+    fun infoMessagesShowInLog(){
```
Comment (migs647) [inline analytics/src/test/java/com/segment/analytics/LoggerTest.kt:54]: Same nit as above

File: analytics/src/test/java/com/segment/analytics/LoggerTest.kt [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                .containsExactly(
+                LogItemBuilder()
+                        .type(Log.INFO)
+                        .msg("some message with an argument")
+                        .build())
+    }
+
+    @Test
+    @Throws
+    fun errorMessagesShowInLog(){
```
Comment (migs647) [inline analytics/src/test/java/com/segment/analytics/LoggerTest.kt:69]: Same nit as above

File: analytics/src/test/java/com/segment/analytics/LoggerTest.kt [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+                LogItemBuilder()
+                        .type(Log.ERROR)
+                        .throwable(throwable)
+                        .msg("some message with an argument")
+                        .build())
+    }
+
+    @Test
+    @Throws
+    fun subLog(){
```
Comment (migs647) [inline analytics/src/test/java/com/segment/analytics/LoggerTest.kt:85]: Same nit as above

File: analytics/src/test/java/com/segment/analytics/LoggerTest.kt [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+        Assertions.assertThat(ShadowLog.getLogs()).containsExactly(
+                LogItemBuilder()
+                        .tag("Analytics-foo")
+                        .type(Log.DEBUG)
+                        .msg("some message with an argument")
+                        .build())
+    }
+
+    class LogItemBuilder{
```
Comment (migs647) [inline analytics/src/test/java/com/segment/analytics/LoggerTest.kt:98]: Same nit as above. `class LogItemBuilder {`

File: analytics/build.gradle [hunk 6]
```diff
@@ -30,7 +31,12 @@ dependencies {
   // with Android.
   //noinspection GradleDependency
   testImplementation 'org.assertj:assertj-core:1.7.1'
+  implementation "androidx.core:core-ktx:+"
```
Comment (egor-n) [inline analytics/build.gradle:34]: Should the dependencies be pinned to a specific version instead? This way there are no surprises when the API changes in the new updates.

Also, it seems like this dependency isn't actually required right now.

**Pull request merged**